### PR TITLE
[Merged by Bors] - chore: Slightly modify the split of `CanonicalEmbedding`

### PR DESCRIPTION
**Pull request by xroblot**
Create a new directory for the split of `CanonicalEmbedding` since more material is coming.

Also add some docstrings.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean [hunk 1]
```diff
@@ -507,5 +507,3 @@ theorem mem_span_fractionalIdealLatticeBasis (x : (E K)) :
   rfl
 
 end integerLattice
-
-end NumberField.mixedEmbedding
```
Comment (Ruben-VandeVelde) [inline Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean:511]: I know it's not necessary, strictly speaking, but I'd keep this line.

**Review by Ruben-VandeVelde — APPROVED** [review]
lgtm, one comment

File: Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean (hunk 1 above)
Comment (xroblot) [reply Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean:511]: Yes. you're right.

File: Mathlib/NumberTheory/NumberField/CanonicalEmbedding/ConvexBody.lean [hunk 2]
```diff
@@ -5,20 +5,33 @@ Authors: Xavier Roblot
...
 
 /-!
-# Convex Bodies
+# Canonical embedding of a number field
 
-This file contains theorems about convex bodies on infinite places of a number field.
+The canonical embedding of a number field `K` of degree `n` is the ring homomorphism
+`K →+* ℂ^n` that sends `x ∈ K` to `(φ_₁(x),...,φ_n(x))` where the `φ_i`'s are the complex
+embeddings of `K`. Note that we do not choose an ordering of the embeddings, but instead map `K`
+into the type `(K →+* ℂ) → ℂ` of `ℂ`-vectors indexed by the complex embeddings.
```
Comment (fpvandoorn) [inline Mathlib/NumberTheory/NumberField/CanonicalEmbedding/ConvexBody.lean:18]: Was this intentionally duplicated from the `Basic` file?

**Review by fpvandoorn — COMMENTED** [review]
LGTM

bors d+

**Comment by xroblot** [comment]
bors +

**Pull request closed without merging**